src/appleseed/CMakeLists.txt

```diff
@@ -115,10 +111,10 @@ set (foundation_image_sources
     foundation/image/iprogressiveimagefilereader.h
     foundation/image/nativedrawing.cpp
     foundation/image/nativedrawing.h
+    foundation/image/oiioimagefilewriter.cpp
```

Comment:
Out of consistency, `OIIOImageFileWriter` should be called `GenericImageFileWriter` (replacing the existing class).

Reply:
It seems that you missed this comment? It's quite important, because the _current_ `GenericImageFileWriter` class is still limited to PNG and OpenEXR files and will throw an exception for all other formats...

Reply:
Done